README.md

```diff
@@ -17,6 +17,16 @@ For details on compatibility, advanced configurations, and add-ons, see [Configu
 
 Examples can be found [here](/bmc-examples/src/main/java/).
 
+You may run any example by invoking the `exec:java` goal and passing appropriate values for `exec.mainClass` and `.exec.arguments` properties,
+for example: `ObjectStorageGetBucketExample` class dwwhich requires 3 argumentsxi which are OCID of the compartment, name of bucket, name of object. This example class can be run as follows:
```

Comment:
Ahh, it seems like there are spelling errors. please remove "dwwhich" and "xi" after arguments. Also, change run to executed in "This example class can be run as follows"